src/rmkit/input/events.cpy

```diff
@@ -133,6 +151,30 @@ namespace input:
               self.lifted = true
 
           break
+        case ABS_MT_WIDTH_MAJOR:
+          slots[slot].width_major = data.value
+          break
+        case ABS_MT_WIDTH_MINOR:
+          slots[slot].width_minor = data.value
+          break
+        case ABS_MT_TOUCH_MAJOR:
+          slots[slot].size_major = data.value
+          break
+        case ABS_MT_TOUCH_MINOR:
+          slots[slot].size_minor = data.value
+          break
+
+    bool is_palm():
+      size := 0
+      for i := 0; i <= MAX_SLOTS; i++:
+        if slots[i].left == 1:
+          size += slots[i].size_major * slots[i].size_minor
```

Comment:
curious to see if this gets triggered with 5+ fingers. I wonder which is more accurate -- sum of all slots, or just the max size of any single slot. Or maybe there's a threshold that works for both? (i.e. sum_size > 1000 || max_size > 150)